packages/ui/exchange-ui-core/src/types/supply.ts

```diff
@@ -0,0 +1,20 @@
+import { SupplyDto, UpdateSupplyDto } from '@energyweb/exchange-client';
+import { IDevice } from '@energyweb/origin-backend-core';
+
+export interface IDeviceWithSupply extends IDevice, Partial<Pick<SupplyDto, 'active' | 'price'>> {
```

Comment:
I think I would not make any binding between IDeviceWithSupply and IDevice, just define the fields you need here, like name and others. This will be useful when we would push data from `local` and other device registries. IDevice is considered old interface that is only compatible with `...device...form` api.